.circleci/config.yml

```diff
@@ -123,6 +123,11 @@ jobs:
           environment:
             NODE_ENV: production
             NODE_OPTIONS: --max_old_space_size=8198
+      - run:
+          command: ./.circleci/scripts/publish-extension.sh
+          environment:
+            NODE_ENV: production
+            NODE_OPTIONS: --max_old_space_size=8198
```

Comment:
this probably ins't necessary, it's only needed for devtools app/kai now I think, most likely because of faker https://discord.com/channels/837138313172353095/1067524003499094046/1067779298213244988